### DISCUSSION TITLE
why couldn't jupyter notebook connect kernel when I openned an example from the idaesx GUI?

### DISCUSSION THREAD

**Discussion by YaxinYangYang** [IDAES/idaes-pse#1309] [Q&A] [answered]
I don't know why jupyter notebook can not connect kernel successfully when I open any example from the IDAES GUI directly, but it can work when I use the command that "jupyter notebook --address"
![image](https://github.com/IDAES/idaes-pse/assets/153571849/e6b7d8b4-eda0-4646-b2de-b122c4008d87)
![image](https://github.com/IDAES/idaes-pse/assets/153571849/5ceb1549-633e-40f4-9c25-9c63300c5700)
Really want to know why! Please!Thank u very much!

**Comment by ksbeattie** [comment, marked as answer]
Did you try creating an entirely new conda environment for this?
